tms/src/main/java/com/netcracker/edu/tms/ui/ProjectRestController.java

```diff
@@ -40,13 +47,22 @@
             return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
         }
 
+        //this block must be before last return line, because in case of invalid addProject function block must not be executed
+        //start of the  block
+
+        if (this.enable.equals("true")) {
```

Comment:
enable flad should be used in MailSerivceImpl logic

Reply:
we send notification !BEFORE! trying to add project to database

imagine the situation when adding project operation failed 
in this case notification is sent anyway, but project does not exist actualy